src/pages/solution/adobe-campaign-classic/tabs/push-payloads.md

```diff
@@ -0,0 +1,161 @@
+| Badge Count | ⛔️ | `adb_n_count` | string | Value to be used on app's badge. |
+| Priority | ⛔️ | `adb_n_priority` | string | Designates the notification priority for Android - [importance](https://developer.android.com/reference/android/app/NotificationChannel#setImportance(int)) for API >= 26 (Android 8.0) or [priority](https://developer.android.com/reference/androidx/core/app/NotificationCompat.Builder#setPriority(int)) for API < 26 |
+| Tag | ⛔️ | `adb_tag` | string | Identifier used to replace existing notifications in the notification drawer. If not specified, each request creates a new notification. If specified and a notification with the same tag is already being shown, the new notification replaces the existing one in the notification drawer. |
+| Sticky | ⛔️ | `adb_sticky` | string | Boolean value represented as a string "true" or "false".<br />When set to false or unset, the notification is automatically dismissed when the user clicks it in the panel. When set to true, the notification persists even when the user clicks it. |
+| Ticker | ⛔️ | `adb_ticker` | string | Sets the "ticker" text, which is sent to accessibility services. Prior to API level 21 (Lollipop), sets the text that is displayed in the status bar when the notification first arrives. |
+| Color - Small Icon | ⛔️ | `adb_clr_icon` | string | Color for the notification's small icon.<br />Represented as six character hex, e.g. `00FF00` |
+| Color - Background | ⛔️ | `adb_clr_bg` | string | Color for notification's background.<br />Represented as six character hex, e.g. `00FF00` |
+
+### Example
+
+Below is a sample of what a payload might look like for a basic notification:
+
+```json
+{
+	"message": {
+		"android": {
+			"collapse_key": "new_message",
+			"priority": "HIGH",
+			"data": {
+				"adb_version": "1",
+                "adb_template_type": "basic",
+                "adb_title": "game request",
+				"adb_body": "shall we play a game?",
+				"adb_sound": "bingBong",
+				"adb_icon": "ic_knight",
+				"adb_n_count": "1",
+				"adb_n_priority": "PRIORITY_LOW",
+				"adb_channel_id": "a3b80ef",
+				"adb_image": "https://pictureofchess.com/board.png",
+				"adb_uri": "https://chess.com/games",
+				"adb_a_type": "WEBURL",
+				"adb_act": "[{\"label\":\"accept\",\"uri\":\"https://chess.com/games/552\",\"type\":\"DEEPLINK\"},{\"label\":\"decline\",\"uri\":\"\",\"type\":\"OPENAPP\"}]",
+				"adb_tag": "24",
+				"adb_sticky": "true",
+				"adb_ticker": "Play a game?",
+				"adb_body_ex": "Bob wants to play a game of chess with you. Click 'accept' to start!",
+				"adb_clr_body": "00EE00",
+				"adb_clr_title": "AABBCC",
+				"adb_clr_icon": "123456",
+				"adb_clr_bg": "000000",
+				"adb_rem_txt": "Remind me",
+				"adb_rem_ts": "1703462400",
+				"customKey": "custom data"
+            }
+		}
+	}
+}
+```
+
+<Variant platform="apns" template="basic" repeat="1"/>
+
+### Coming soon!
+
+<Variant platform="fcm" template="carousel" repeat="6"/>
+
+### Properties
```

Comment:
is this section a duplicate of the one starting at https://github.com/AdobeDocs/aep-mobile-sdkdocs/pull/477/files#diff-94bef8f79c929b7980d99f27b255e44d1cb7753f3322b0c2d462f5b5fb8243caR20?